stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -106,6 +112,9 @@ extension ${Self}: CustomStringConvertible {
 ${Availability(bits)}
 extension ${Self}: CustomDebugStringConvertible {
   /// A textual representation of the value, suitable for debugging.
+  ///
+  /// This property has the same value as the `description` property, except
+  /// that NaN values are printed in an extended format.
```

Comment:
I don’t know if we want to guarantee this format for all time (@steventyrone?). If so, I would describe what the “extended” format is (something like: “For debugging purposes, the representation includes any NaN payload and distinguishes between quiet and signaling NaN; for all other values, it is identical to the value’s `description`.”).

Reply:
That makes sense, though I'd prefer to mention the non-NaN case first (since that's probably more common):

> This property is identical to the `description` property for any value that is not a NaN.  For improved debugging, the `debugDescription` of a NaN value also includes any NaN payload, quiet/signaling bits, and sign.